src/main/java/org/apache/sling/resourceresolver/impl/mapping/MapEntries.java

```diff
@@ -18,34 +18,6 @@
  */
 package org.apache.sling.resourceresolver.impl.mapping;
 
-import org.apache.commons.lang3.StringUtils;
```

Comment:
The PR is reordering imports - it would be good to avoid this to see the real changes

Reply:
will do